modules/gitops-argo-rollouts-cli-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI overview
+
+Argo Rollouts CLI is an optional plugin that allows you to handle and monitor Argo Rollouts resources directly through the command line interface instead of using {OCP} web console or the `oc` command. 
+
+The Argo Rollouts CLI plugin enables you to perform many actions, including the following actions:
+
+* Make changes to an Argo Rollouts image.
+* Monitor the progress of an Argo Rollouts promotion.
+* Proceed with the promotion steps in a canary deployment.
+* Terminate a failed Argo Rollouts deployment.
+
+The Argo Rollouts CLI plugin is an `oc kubectl` plugin, and integrates directly with the `oc kubectl` commands.
```

Comment:
```suggestion
The Argo Rollouts CLI plugin directly integrates with `oc` and `kubectl` commands.
```

Reply:
This makes it more concise while still conveying the same meaning :+1:

Reply:
Fixed.